.github/ISSUE_TEMPLATE/package_build_request.yml

```diff
@@ -0,0 +1,105 @@
+name: Package build request
+description: Request a package to be built (e.g. new, updates, and rebuilds).
+labels: ['package_request']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please use this form to submit a request for a package to be built.
+
+        We kindly ask that you take the time to:
+          - Search Anaconda.org ([anaconda](https://anaconda.org/anaconda) channel) to verify whether the package already exists in our channels.
```

Comment:
I don't think we want to point them at anaconda.org/anaconda - it would be better to point them at https://repo.anaconda.com/pkgs/main/ and https://repo.anaconda.com/pkgs/r/

Reply:
Done. Thank you!